app/models/message.js

```diff
@@ -1,35 +1,35 @@
-import Ember from 'ember';
-import DS from 'ember-data';
+import Ember from "ember";
+import DS from "ember-data";
 var attr = DS.attr,
-belongsTo = DS.belongsTo;
+  belongsTo = DS.belongsTo;
 
 export default DS.Model.extend({
   session: Ember.inject.service(),
-  body: attr('string'),
-  isPrivate: attr('boolean'),
-  createdAt: attr('date'),
-  updatedAt: attr('date'),
-  senderId: attr('number'),
-  state: attr('string', {
-    defaultValue: 'read'
+  body: attr("string"),
+  isPrivate: attr("boolean"),
+  createdAt: attr("date"),
+  updatedAt: attr("date"),
+  senderId: attr("number"),
+  state: attr("string", {
+    defaultValue: "read"
   }),
-  sender: belongsTo('sender', {
+  sender: belongsTo("user", {
     async: false
   }),
-  designation: belongsTo('designation', { async: false }),
+  designation: belongsTo("designation", { async: false }),
 
-  myMessage: Ember.computed(function () {
-    return this.get("sender.id") === this.get('session.currentUser.id');
+  myMessage: Ember.computed(function() {
```

Comment:
You are returning values on `sender` property, then you should add `sender` on the CP values list.
```
Ember.computed('sender', function(){})
```
Add similar change for other CP methods too.

Reply:
sure, thanks